### PR TITLE
Update Python versions.

### PR DESCRIPTION
**Pull request by jugmac00**
- Dropped support for Python 3.4.
- Added support for Python 3.8.

modified:   .travis.yml
modified:   CHANGES.txt
modified:   setup.py
modified:   tox.ini

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,10 +47,10 @@ def read(*rnames):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8-pre',
```
Comment (mgedmin) [inline setup.py]: I do not see Programming Language :: Python :: 3.8-pre on https://pypi.org/classifiers/

**Review by mgedmin — CHANGES_REQUESTED** [review]
LGTM except for the bogus PyPI classifier.

File: setup.py (hunk 1 above)
Comment (jugmac00) [reply setup.py]: Just updated it. Thanks for the review.

**Review by mgedmin — APPROVED** [review]
:+1:

**Pull request merged**